src/test/scala/io/iohk/ethereum/jsonrpc/CheckpointingJRCSpec.scala

```diff
@@ -0,0 +1,102 @@
+package io.iohk.ethereum.jsonrpc
+
+import io.iohk.ethereum.crypto.ECDSASignature
+import io.iohk.ethereum.jsonrpc.CheckpointingService._
+import io.iohk.ethereum.jsonrpc.JsonRpcController.JsonRpcConfig
+import io.iohk.ethereum.utils.{ByteStringUtils, Config}
+import io.iohk.ethereum.{Fixtures, NormalPatience}
+import org.json4s.JsonAST._
+import org.json4s.JsonDSL._
+import org.scalamock.scalatest.MockFactory
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+import scala.concurrent.Future
+
+class CheckpointingJRCSpec
+  extends AnyFlatSpec
+    with Matchers
+    with MockFactory
+    with ScalaFutures
+    with NormalPatience
+    with JRCMatchers
+    with JsonMethodsImplicits {
+
+  import Req._
+
+  "CheckpointingJRC" should "getLatestBlock" in new TestSetup {
```

Comment:
Should we maybe add tests for the non happy cases as when the signatures can't be decoded?